backend-lib/src/main/rust/lib.rs

```diff
@@ -1379,17 +1468,16 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_sh
     output_params: JString<'_>,
     network_id: jint,
     use_zip317_fees: jboolean,
-) -> jlong {
+) -> jbyteArray {
     let res = panic::catch_unwind(|| {
         let network = parse_network(network_id as u32)?;
-        let db_data = wallet_db(&env, network, db_data)?;
-        let mut db_data = db_data.get_update_ops()?;
+        let mut db_data = wallet_db(&env, network, db_data)?;
         let usk = decode_usk(&env, usk)?;
         let memo_bytes = env.convert_byte_array(memo).unwrap();
         let spend_params = utils::java_string_to_rust(&env, spend_params);
         let output_params = utils::java_string_to_rust(&env, output_params);
 
-        let min_confirmations = 0;
+        let min_confirmations = NonZeroU32::new(1).unwrap();
```

Comment:
Same bug: in the iOS SDK I use `target` instead of `anchor` to re-enable zero-conf.

Really, what we should do is revert the `NonZeroU32` change for the `shield_transparent_funds` API (or implement https://github.com/zcash/librustzcash/issues/873). That doesn't block the RC.

Reply:
@str4d Same case as above.